### ISSUE TITLE
wrong placement

### ISSUE THREAD

**Issue by Jiehong** [mhayashi1120/Emacs-langtool#5]
When checking a sentence, the placement of the error and of the correction is wrong.

For example:
First sentence. For instance, most of people are chinese.
However, most of people are not old.

In the first one, the " most of peopl" will be highlighted and the correction will lead to " For instance,most of the peoplee"… which is just bad.
But the second one will be correct. This only appear if one or more sentence are contained in the line.

Eventually, the last example:
First sencence. Second sentence but a bit longer. For instance, most of people are using Emacs.

In this case, the last sentence is matched as "t most of peop".

To sum up:
The position of the mistake is good in the first sentence of the line.
The position of the mistakes in sentences after the first one will shift on the right by 1 caracter on the left for each sentence.
